tests/gaussian/test_gaussian.py

```diff
@@ -0,0 +1,145 @@
+            self.assertEqual(self.job.input['settings'], {'SCF':['MaxCycle=90']})
+
+
+        def test_calc_minimize(self):
+            self.job.calc_minimize(
+                electronic_steps=80,
+                ionic_steps=90,
+                algorithm=None,
+                ionic_forces='tight',
+            )
+            self.assertEqual(self.job.input['jobtype'], 'opt(MaxCycles=90,tight)')
+            self.assertEqual(self.job.input['settings'], {'SCF':['MaxCycle=80']})
+
+        def test_set_structure(self):
+            self.assertEqual(self.job.structure, None)
+            atoms = Pr.create_atoms(elements=['H', 'H', 'O'], positions=[[0.,0.,0.], [1.,0.,0.], [0.,1.,0.]])
+            self.job.structure = atoms
+            self.assertEqual(self.job.structure, atoms)
+            self.job.structure = None
+            self.assertEqual(self.job.structure, None)
+            self.job.structure = atoms
+            self.assertEqual(self.job.structure, atoms)
+
+
+        def test_run_complete(self):
+            self.job_complete.input['lot'] = 'B3LYP'
+            self.job_complete.input['basis_set'] = '6-31+G(d)'
+            self.job_complete.calc_static()
+            file_directory = posixpath.join(
+                self.execution_path, '../static/gaussian_test_files/sp'
+            )
+            self.job_complete.restart_file_list.append(
+                posixpath.join(file_directory, "input.log")
+            )
+            self.job_complete.restart_file_list.append(
+                posixpath.join(file_directory, "input.fchk")
+            )
+            self.job_complete.run(run_mode="manual")
+            self.job_complete.status.collect = True
+            self.job_complete.run()
+            nodes = [
+                "positions",
+                "energy_tot",
+            ]
+            with self.job_complete.project_hdf5.open("output/generic") as h_gen:
+                hdf_nodes = h_gen.list_nodes()
+                self.assertTrue(all([node in hdf_nodes for node in nodes]))
+
+            nodes = [
+                "charges",
+                "dipole",
+                "masses",
+                "numbers",
+                "positions",
+            ]
+            with self.job_complete.project_hdf5.open("output/structure") as h_gen:
+                hdf_nodes = h_gen.list_nodes()
+                self.assertTrue(all([node in hdf_nodes for node in nodes]))
+
+            nodes = [
+                "alpha_orbital_e",
+                "beta_orbital_e",
+                "n_alpha_electrons",
+                "n_basis_functions",
+                "n_beta_electrons",
+                "n_electrons",
+                "scf_density",
+                "spin_scf_density",
+            ]
+            with self.job_complete.project_hdf5.open("output/structure/dft") as h_gen:
+                hdf_nodes = h_gen.list_nodes()
+                self.assertTrue(all([node in hdf_nodes for node in nodes]))
```

Comment:
I have the feeling the tests are not executed by our automated testing system, can you add: 

```
if __name__ == "__main__":
    unittest.main()
```

At the very end of the test file? Currently I expect the tests to fail, as the requirements are not installed in the test environment.

Reply:
I already collected the requirements in https://github.com/pyiron/pyiron/pull/548